__docs__/wonder-blocks-form/text-area.stories.tsx

```diff
@@ -0,0 +1,444 @@
+import * as React from "react";
+import type {Meta, StoryObj} from "@storybook/react";
+
+import {StyleSheet} from "aphrodite";
+import {TextArea} from "@khanacademy/wonder-blocks-form";
+import packageConfig from "../../packages/wonder-blocks-form/package.json";
+
+import ComponentInfo from "../../.storybook/components/component-info";
+import {color, spacing} from "@khanacademy/wonder-blocks-tokens";
+import Button from "@khanacademy/wonder-blocks-button";
+import {LabelSmall, LabelLarge} from "@khanacademy/wonder-blocks-typography";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {View} from "@khanacademy/wonder-blocks-core";
+
+import TextAreaArgTypes from "./text-area.argtypes";
+
+/**
+ * A TextArea is an element used to accept text from the user.
+ */
+export default {
+    title: "Packages / Form / TextArea",
+    component: TextArea,
+    parameters: {
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+        a11y: {
```

Comment:
There is one remaining Storybook a11y warning, any thoughts on how to resolve it? There doesn't seem to be another element obscuring the textarea 🤔 

<img width="1423" alt="image" src="https://github.com/user-attachments/assets/ba80d0f8-1ae0-467a-a7f3-109dfcb408af">

When I open the Story in a new tab on it's own and use axe DevTools, this contrast warning isn't there:
<img width="1718" alt="image" src="https://github.com/user-attachments/assets/13306f98-9c03-4e6d-9e1e-1234705a9fe4">


I've left the Storybook a11y warning there for now since it is marked as "Incomplete" rather than "Violation". Let me know if you have thoughts on this!

Reply:
Tried changing the background color and even hardcoding the border color to black (since it for sure has a high contrast with black) and was still getting this warning! I also rendered a basic unstyled `<textarea>` in a story and the warning is still there (with and without the resize control).  I also noticed this warning shows in all browsers except for Firefox. 🤷 Since the TextArea styling is following the same styling as TextField, I will leave this as is!